### PR TITLE
[UBS] - #4939-convert-delete-endpoint-to-patch

### PR DESCRIPTION
**Pull request by Markiro1**
1) add new ErrorMessage for check deactivate courier
2) add root for endpoint deactivateCourier() method in SecurityConfig
3) rename endpoint in SuperAdminController
4) change return type deactivateCourier() from void to CourierDto 
5) implement deactivateCourier() method in SuperAdminServiceImpl
6) add test for deactivateCourier() method

### CODE REVIEW

**Review by JJJazl — APPROVED** [review]

File: core/src/test/java/greencity/controller/SuperAdminControllerTest.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import greencity.exceptions.tariff.TariffAlreadyExistsException;
 import greencity.filters.TariffsInfoFilterCriteria;
 import greencity.service.SuperAdminService;
+import liquibase.pro.packaged.E;
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/SuperAdminControllerTest.java]: unused import

File: core/src/test/java/greencity/controller/SuperAdminControllerTest.java [hunk 2]
```diff
@@ -416,6 +417,11 @@ void deactivateTariffTest() {
             .andExpect(status().isOk());
     }
 
+    @Test
+    void deactivateCourier() throws Exception {
+        mockMvc.perform(patch(ubsLink + "/deactivateCourier/{id}", 1L)).andExpect(status().isOk());
```
Comment (RomaMocherniuk) [inline core/src/test/java/greencity/controller/SuperAdminControllerTest.java:421]: please verify that `superAdminService.deactivateCourier(id)` was called with the expected `id`

File: service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java [hunk 3]
```diff
@@ -487,25 +487,34 @@ void includeBagException2Test() {
...
+    }
 
-        verify(courierRepository).findById(1L);
-        verify(courierRepository).save(courier);
+    @Test
+    void deactivateCourierThrowBadRequestException() {
+        Courier courier = getCourier();
+        when(courierRepository.findById(anyLong())).thenReturn(Optional.of(courier));
+        courier.setCourierStatus(CourierStatus.DEACTIVATED);
+        assertThrows(BadRequestException.class, () -> superAdminService.deactivateCourier(anyLong()));
```
Comment (RomaMocherniuk) [inline service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java]: please add `verify`

File: service-api/src/main/java/greencity/constant/ErrorMessage.java [hunk 4]
```diff
@@ -67,6 +67,7 @@ public final class ErrorMessage {
     public static final String LOCATION_STATUS_IS_ALREADY_EXIST =
         "Current location already has status that's you wanna chose";
     public static final String COURIER_IS_NOT_FOUND_BY_ID = "Couldn't found courier by id: ";
+    public static final String CANNOT_DEACTIVATE_COURIER = "courier is already deactivated with id: ";
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/constant/ErrorMessage.java]: Should the message start with a capital letter?

**Pull request closed without merging**